source/patterns/@aws-solutions-constructs/core/lib/dynamodb-table-helper.ts

```diff
@@ -60,6 +76,18 @@ export function buildDynamoDBTable(scope: Construct, props: BuildDynamoDBTablePr
   }
 }
 
+export function buildDynamoDBTableInterface(scope: Construct, props: BuildDynamoDBTableInterfaceProps): [dynamodb.ITable, dynamodb.Table?] {
+  // Conditional DynamoDB Table creation
+  if (!props.existingTableInterface) {
+    // Set the default props for DynamoDB table
+    const dynamoTableProps = consolidateProps(DefaultTableProps, props.dynamoTableProps);
```

Comment:
change name to consolidatedTableProps - having the same name as the props attribute is confusing. Plus, in the context of this function, "dynamo" adds no value.